Build A GitHub Repository Tracker With Directus Flow/github_repository_tracker.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "Build A GitHub Repository Tracker With Directus Flow"
+description: "Learn how to keep track of changes in a GitHub repository in your Directus project."
+author:
+  name: "Erinle Samuel"
+  avatar_file_name: "profile-picture.jpg"
+---
+
+## Introduction
+
+Keeping tabs on repository activities like issues, comments and pull requests helps to ensure timely responses and manage workflow effectively during project development.
+In this tutorial, you will learn how to track changes in a GitHub repository within your Directus project using Directus flow. By the end of this article, you will have created a repository tracker that sends alerts via email and in-app notifications.
+
+## Before You Start
+
+You will need:
+
+- A Directus project - Create a project by following this [quickstart guide](https://docs.directus.io/getting-started/quickstart.html)
+
+- [A GitHub Account](https://github.com/signup)
+
+:::warning
+
+To follow this tutorial, ensure you have admin access to the GitHub repository you wish to track.
+
+Note that you won't be able to follow along with this tutorial if your Directus project is running on localhost. GitHub cannot send data to [http://localhost:8055](http://localhost:8055) because it's not a remote location.
+
+For ease, use [Directus Cloud](https://directus.cloud/) or deploy your project to a remote server; you can refer to [this article](https://docs.directus.io/blog/deploying-directus-to-aws-ec2-with-docker.html) for guidance.
+
+:::
+
+## Creating The Flow
+
+In your Directus project, navigate to **Settings** => **Flows** and create a Flow. In the Trigger Setup section, select **Webhook Trigger** so we can listen to events from GitHub. Ensure the Webhook Trigger is set to use the `POST` method with Asynchronous mode enabled. Save the Flow and copy the Trigger URL.
+
+![A screenshot of the Flow demonstrating where to find the Trigger URL](flow_setup.png)
+
+## Setting Up a Webhook on GitHub
+
+GitHub allows developers to create webhooks for both repositories and organizations using the same process. To create a webhook, navigate to the repository or organization you want to track. Go to the **Settings** tab and select **Webhooks** from the menu.
+
+![A screenshot of GitHub setting => Webhooks page](github_webhook_page.png)
+
+Click on the **Add Webhook** button. Next, paste the Trigger URL from Directus flow into the `Payload URL` field and change the content type to `application/json`.
+
+Select the event(s) that should trigger the webhook. For this tutorial, choose the `Let me select individual events` option and toggle these events:
+
+- Issues,
+- Issue comments, and
+- Pull requests.
+
+Save your settings by clicking the **Add Webhook** button. GitHub will send a ping payload to your Directus project. Confirm the webhook is successful by checking the Activity Logs section.
+
+![A screenshot of Activity Logs section on](github_ping.png)
```

Comment:
```suggestion
## Setting Up a Webhook on GitHub

GitHub allows developers to create webhooks for both repositories and organizations. A webhook is a web request to an external URL (in this case, Directus) that will be immediately triggered when an event happens such as a new issue being opened in a repository. 

To create a webhook, navigate to the repository or organization you want to track. Go to the **Settings** tab and select **Webhooks** from the menu.

Click on the **Add Webhook** button. Next, paste the Trigger URL from Directus flow into the `Payload URL` field and change the content type to `application/json`.

Select the events that should trigger the webhook. For this tutorial, choose the `Let me select individual events` option and toggle these events:

- Issues
- Issue comments
- Pull requests

Save your settings by clicking the **Add Webhook** button. GitHub will send a ping payload to your Directus project. Confirm the webhook is successful by checking the Activity Logs section in your Flow.

![A screenshot of Activity Logs section in the Flow editor](github_ping.png)
```

A whole bunch of small changes here:
1. Explain what webhooks are
2. Link them back to the project
3. Remove the image of GitHub as it doesn't serve much value